README.md

```diff
@@ -1,2 +1,18 @@
 # tilelive-AsMVT
+
 Tilelive module for generating Mapbox Vector Tiles from a Postgres DB using ST_AsMVT.
+
+Requires a postgres function that takes 3 int arguments(zoom, x, y) and returns vector tile data to exist.
+
+## Example source url:
+
+`asmvt:///?host=localhost&user=postgres&database=osm&port=15432&function=gettile`
+
+## Query parameters:
+
+- `user` - postgres user, defaults to `PGUSER` environment variable if not provided.
+- `host` - postgres host, defaults to `PGHOST` environment variable if not provided.
+- `database` - database name, defaults to `PGDATABASE` environment variable if not provided.
+- `password` - postgres password, defaults to `PGPASSWORD` environment variable if not provided.
+- `port` - postgres port, defaults to `PGPORT` environment variable if not provided.
+- `function` - name of database function that returns tile data, defaults to `gettile` if not provided.
```

Comment:
Since `gettile` isn't a default function, I'd consider making this parameter required and loudly exiting if it is not provided.